.github/workflows/main.yml

```diff
@@ -17,10 +17,10 @@ jobs:
    - uses: actions/checkout@v1
    - name: Build docker image
      run: |
-       docker login docker.pkg.github.com -u ${GITHUB_REPOSITORY} -p "${{ secrets.GITHUB_TOKEN }}"
-       docker pull --disable-content-trust docker.pkg.github.com/kotekan/kotekan/kotekan:latest
-       docker build --cache-from=rocm/dev-ubuntu-18.04,docker.pkg.github.com/kotekan/kotekan/kotekan:latest -f tools/iwyu/docker/Dockerfile -t docker.pkg.github.com/kotekan/kotekan/kotekan:latest .
-       docker push docker.pkg.github.com/kotekan/kotekan/kotekan:latest
-   - name: Run iwyu
-     run: |
-       docker run --rm docker.pkg.github.com/kotekan/kotekan/kotekan:latest /code/tools/iwyu/docker/iwyu.sh
+       docker login docker.pkg.github.com -u ${GITHUB_REPOSITORY} -p "${{ secrets.GITHUB_TOKEN }}" | while read line ; do echo "$(date)| $line"; done;
+       docker pull --disable-content-trust docker.pkg.github.com/kotekan/kotekan/kotekan:latest | while read line ; do echo "$(date)| $line"; done;
+       docker build --cache-from=rocm/dev-ubuntu-18.04,docker.pkg.github.com/kotekan/kotekan/kotekan:latest -f tools/iwyu/docker/Dockerfile -t docker.pkg.github.com/kotekan/kotekan/kotekan:latest . | while read line ; do echo "$(date)| $line"; done;
+       docker push docker.pkg.github.com/kotekan/kotekan/kotekan:latest | while read line ; do echo "$(date)| $line"; done;
+  #  - name: Run iwyu
```

Comment:
We could still do this, just remove `develop` from the branches above and limit it on PRs/pushes towards master as you suggested.